catalog/spatial/registry/registry-schema-bindings/pom.xml

```diff
@@ -135,6 +164,7 @@
                             oasis.names.tc.ebxml_regrep.xsd.rim._3
                             ,net.opengis.ogc
                             ,net.opengis.cat.wrs.v_1_0_2
+                            ,org.codice.ddf.registry.schemabindings
```

Comment:
Nit: Mind moving all the `,` at the end of the previous line instead of the beginning?